packages/driver/src/multi-domain/cypress.ts

```diff
@@ -73,9 +74,32 @@ const onBeforeAppWindowLoad = (Cypress: Cypress.Cypress, cy: $Cy) => (autWindow:
   cy.overrides.wrapNativeMethods(autWindow)
   // TODO: DRY this up with the mostly-the-same code in src/cypress/cy.js
   bindToListeners(autWindow, {
-    // TODO: implement this once there's a better way to forward
-    // messages to the top frame
-    onError: () => () => undefined,
+    onError: (handlerType) => {
+      return (event) => {
+        const { originalErr, err, promise } = $errUtils.errorFromUncaughtEvent(handlerType, event) as ErrorFromProjectRejectionEvent
```

Comment:
This functionality is somewhat duplicated from what happens in the [primary domain](https://github.com/cypress-io/cypress/blob/d8fa85d3dc9888c3fe65af9b4fb8291939f5675d/packages/driver/src/cypress/cy.ts#L1077-L1093), but is slightly different. Figured I would punt on this for now as a abstraction pattern for these events isn't yet clear to me.